exchange/exchange_test.go

```diff
@@ -526,7 +526,8 @@ func newExtRequest() openrtb_ext.ExtRequest {
 		},
 	}
 
-	brandCat := openrtb_ext.ExtIncludeBrandCategory{PrimaryAdServer: 1, WithCategory: true}
+	translateCat := true
```

Comment:
Consider renaming to `translateCategories` to match the naming of exchange.go.

Reply:
Updated to `translateCategories`